src/MLJDecisionTreeInterface.jl

```diff
@@ -249,7 +341,16 @@ function MMI.fit(m::RandomForestRegressor, verbosity::Int, X, y)
                               m.min_purity_increase,
                               rng=m.rng)
     cache  = nothing
-    report = NamedTuple()
+
+    if m.feature_importance != :none
+        fi = feature_importance_options[m.feature_importance](forest)
+        fi_pairs = collect(Dict(zip(features, fi)))
+        # sort descending
+        sort!(fi_pairs, by= x->-x[2])
+        report = (feature_importances=fi_pairs,)
+    else
+        report = NamedTuple()
+    end
```

Comment:
put computation for feature importances in `MMI.feature_importances` method. See comments above